css/cssom-view/scrollIntoView-vertical-rl-writing-mode.html

```diff
@@ -62,14 +65,40 @@
 var box_width = target.offsetWidth;
 var box_height = target.offsetHeight;
 
-var expectedX = [ ((2*box_width)-scroller_width)+scrollbar_width, ((3*box_width - scroller_width)/2)+ (scrollbar_width/2), box_width ];
-var expectedY = [ box_height, ((3*box_height - scroller_height)/2) + (scrollbar_width/2), ((2*box_height)-scroller_height) + scrollbar_width ];
+var expectedX;
+// If scroll bar is on the left side, scroller.scrollLeft won't bigger than 0,
+scroller.scrollLeft = scrollbar_width;
+if (scroller.scrollLeft == 0) {
+  expectedX = {
+    blockStart: -box_width,
+    blockCenter: -(((3*box_width - scroller_width)/2) + (scrollbar_width/2)),
+    blockEnd: -(((2*box_width) - scroller_width) + scrollbar_width),
+  };
+} else {
+  expectedX = {
+    blockStart: -(box_width - scrollbar_width),
+    blockCenter: -(((3*box_width - scroller_width)/2) + (scrollbar_width/2) - scrollbar_width),
+    blockEnd: -((2*box_width) - scroller_width),
+  };
+}
+
+var expectedY = {
+  inlineStart: box_height,
+  inlineCenter: ((3*box_height - scroller_height)/2) + (scrollbar_width/2),
+  inlineEnd: ((2*box_height) - scroller_height) + scrollbar_width,
+};
 
 // As browsers differ in the meaning of scrollLeft when
 // in a right-to-left mode, we adjust the expectation
 // to match the semantics of scrollLeft.
```

Comment:
Can you please update the comment:

"In vertical-rl mode, the scroll x coordinate should be nonpositive per the the spec. Some browsers make it nonnegative, so we adjust the expectation.